modules/ossm-config-control-plane-infrastructure-node-cli.adoc

```diff
@@ -3,10 +3,10 @@
 // * service_mesh/v2x/ossm-deployment-models.adoc
 
 :_content-type: PROCEDURE
-[id="ossm-config-control-plane-infrastructure-node_{context}"]
-= Configuring all {SMProductShortName} control plane components to run on infrastructure nodes
+[id="ossm-config-control-plane-infrastructure-node-cli_{context}"]
+= Configuring all control plane components to run on infrastructure nodes with the CLI
 
-This task should only be performed if all of the components deployed by the {SMProductShortName} control plane (including Istiod, Ingress Gateway, and Egress Gateway) along with optional elements (such as Prometheus, Grafana, and Distributed Tracing) are running on infrastructure nodes. 
+You can configure all of the {SMProductShortName} control plane components to run on infrastructure nodes using the CLI. This task should only be performed if all of the components deployed by the {SMProductShortName} control plane, including Istiod, Ingress Gateway, and Egress Gateway, along with optional elements, such as Prometheus, Grafana, and Distributed Tracing, are running on infrastructure nodes. 
 
 If the control plane runs on a worker node, skip this task.
```

Comment:
Similar thing here. The purpose of this task is to move the control plane from worker nodes to infra nodes, so the sentence doesn't make sense to me.

Reply:
Done